examples/tutorials/self-paced-training/welcome.ipynb

```diff
@@ -22,18 +22,20 @@
    "metadata": {},
    "source": [
     "# Course Goals\n",
-    "In this course, our primary goal is to **introduce the fundamentals of NVIDIA FLARE to researchers, scientists, and developers**.\n",
+    "In this course, our primary goal is to **introduce the fundamentals of federated learning with NVIDIA FLARE**.\n",
     "\n",
     "This five-part, 12-chapter course is designed to equip participants with the knowledge and skills to develop and deploy federated learning applications using NVIDIA FLARE.\n",
     "\n",
-    "**Part 1-3 (Chapter 1-6): Practial Federated Learning with NVIDIA FLARE - Fundamentals and System Architecture**\n",
+    "**Part 1-3 (Chapter 1-6): Decentralized AI at Scale with NVIDIA FLARE**\n",
```

Comment:
If we want new sentence, need to say AI means Artificial Intelligence.
why do we want new sentence?